helmfile/charts/networkpolicy-common/templates/coredns/allow-coredns.yaml

```diff
@@ -0,0 +1,49 @@
+{{ if .Values.coredns.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-coredns
+spec:
+  podSelector:
+    matchLabels:
+      k8s-app: kube-dns
+  policyTypes:
+    - Egress
+    - Ingress
+  ingress:
+    - ports:
```

Comment:
@Xartos Could this be limited to local-dns? and then exgress for local dns to coredns?

Reply:
Since local-dns is using hostnet it isn't possible